### PR TITLE
Update list of branches for myget publishing

### PR DESCRIPTION
**Pull request by shyamnamboodiripad**
[Infrastructure-only change]

Remove stale branches and add dev15.3.x.

### CODE REVIEW

File: src/Tools/MicroBuild/publish-assets.ps1 [hunk 1]
```diff
@@ -37,13 +37,11 @@ try
...
-        "dev15.0.x" { } 
-        "dev15.1.x" { } 
-        "dev15.2.x" { } 
-        "dev15.3-preview1" { } 
-        "dev15.3-preview2" { }
-        "master" { } 
-        "post-dev15" { } 
+        "dev15.0.x" { }
+        "dev15.1.x" { }
+        "dev15.2.x" { }
```
Comment (shyamnamboodiripad) [inline src/Tools/MicroBuild/publish-assets.ps1]: Can the above ones be deleted now / are we keeping them around for servicing?

**Comment by shyamnamboodiripad** [comment]
@jasonmalinowski @dotnet/roslyn-infrastructure

File: src/Tools/MicroBuild/publish-assets.ps1 (hunk 1 above)
Comment (jasonmalinowski) [inline src/Tools/MicroBuild/publish-assets.ps1]: Yes, you can clean up the old ones. What matters here is the content of the file in the branch itself.

**Comment by shyamnamboodiripad** [comment]
@jasonmalinowski @tannergooding I am guessing there is no equivalent for this file in the ProjectSystem repo - but please ping me if there is. I am pretty sure there isn't one for LUT.

**Pull request merged**